### PR TITLE
Closes #284: Create aff matching module, model and convert the imported organizations

### LINKED ISSUE #284: Create aff matching module, model and convert the imported organizations
E.g. remove unnecessary spaces, change the case to lowercase.

### PR DESCRIPTION
**Pull request by lukdumi**

### CODE REVIEW

File: iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchOrganizerConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+import eu.dnetlib.iis.common.string.StringNormalizer;
+import eu.dnetlib.iis.importer.schemas.Organization;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchOrganization;
+
+/**
+* @author Łukasz Dumiszewski
+*/
+
+public class AffMatchOrganizerConverterTest {
```
Comment (madryk) [inline iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchOrganizerConverterTest.java]: AffMatchOrganiz**ation**ConverterTest

File: iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchOrganizerConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    
+    
+    //------------------------ PRIVATE --------------------------
+    
+    private Organization createOrganization() {
+        
+        Organization org = new Organization();
+        
+        org.setId("ABC123");
+        org.setName("Interdyscyplinary Centre");
```
Comment (madryk) [inline iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchOrganizerConverterTest.java]: A little weird mix of polish and english: `Interdyscyplinary`
`interdisciplinary` would be more correct

File: iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchOrganizerConverterTest.java (hunk 1 above)
Comment (lukdumi) [inline iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchOrganizerConverterTest.java]: :+1:

File: iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchOrganizerConverterTest.java (hunk 2 above)
Comment (lukdumi) [inline iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchOrganizerConverterTest.java]: :+1:

**Pull request closed without merging**